src/browser/web_storage.rs

```diff
@@ -200,7 +223,7 @@ mod tests {
         LocalStorage::clear().unwrap();
 
         let value: String = {
-            LocalStorage::insert("a_key", "a_value").unwrap();
+            LocalStorage::insert(&"a_key", &"a_value").unwrap();
```

Comment:
Are references necessary?

Reply:
https://github.com/cloudflare/serde-wasm-bindgen/pull/18 just got merged, but it's not yet released. Once released, I can add back the `?Sized` "constraint" on all the functions I had to take it off of, and then the references here (and elsewhere) can be reverted.

Reply:
FWIW you can just hide this as an implementation detail for now, either by always using `&` before calling `to_value` or by creating `Serializer` explicitly. It will be easier to update it that way in future too.